src/Perl6/Pod.nqp

```diff
@@ -1665,6 +1665,261 @@ class Perl6::Pod {
 
     } # end sub table
 
+    # all-lowercase HTML5 character entities, derived from
+    # https://github.com/w3c/html/blob/master/entities.json
+    my %entities;
+    # NQP doesn't support assignment of entire hashes :(
+    # so we have to iterate
```

Comment:
We actually don't have to. `my %entitities := nqp::hash('aacute',[225],'aacute',[225], ... );`

Reply:
Let's start with reverting the offending commit, and then look at improvements later?

Reply:
Ok. I didn't notice it's a revert at the first glance.